src/PackageCompiler.jl

```diff
@@ -504,6 +516,9 @@ compiler (can also include extra arguments to the compiler, like `-g`).
    transitive dependencies into the sysimage. This only makes a difference if some
    packages do not load all their dependencies when themselves are loaded. Defaults to `true`.
 
+- `import_into_main::Bool`: If `true`, import all packages from the sysimage into `Main`.
```

Comment:
Is this true? I thought it was just the packages in the `[deps]` section of the project file.

Reply:
No, I believe only deps from the project you're creating a sysimage of go in 
```suggestion
- `import_into_main::Bool`: If `true`, import all packages from `packages` into `Main`.
```

This is more accurate I guess